test/Serialization/load-invalid-arch.swift.gyb.swift

```diff
@@ -0,0 +1,20 @@
+
+// RUN: %empty-directory(%t)
+// RUN: mkdir %t/new_module.swiftmodule
+// RUN: touch %t/new_module.swiftmodule/arm64.swiftmodule
+// RUN: touch %t/new_module.swiftmodule/powerpc64.swiftmodule
+// RUN: touch %t/new_module.swiftmodule/arm64.swiftdoc
+// RUN: touch %t/new_module.swiftmodule/powerpc64.swiftdoc
+// RUN: %gyb -DTARGET_ARCHITECTURE=%target-cpu %s -o %t/load-invalid-arc.swift
+// RUN: %target-swift-frontend %t/load-invalid-arc.swift -typecheck -I %t -verify -show-diagnostics-after-fatal
+
+// RUN: %empty-directory(%t)
+// RUN: mkdir -p %t/new_module.framework/Modules/new_module.swiftmodule/
+// RUN: touch %t/new_module.framework/Modules/new_module.swiftmodule/powerpc64.swiftmodule
+// RUN: touch %t/new_module.framework/Modules/new_module.swiftmodule/arm64.swiftmodule
+// RUN: %gyb -DTARGET_ARCHITECTURE=%target-cpu %s -o %t/load-invalid-arc.swift
+// RUN: %target-swift-frontend %t/load-invalid-arc.swift -F %t -typecheck -verify -show-diagnostics-after-fatal
+
+import new_module // expected-error {{no such module 'new_module'}} expected-error {{could not find module 'new_module' for architecture '${TARGET_ARCHITECTURE}' found: powerpc64, arm64}}
```

Comment:
This seems reasonable. Another approach would be to give up on -verify diagnostics and instead pipe the output into [`%FileCheck`](https://llvm.org/docs/CommandGuide/FileCheck.html).

One note, though: we actually *do* run these tests for arm64 targets too, and someone could try to run them for powerpc64. It might be better to use obviously invalid architecture names instead.